src/start/exceptions.md

```diff
@@ -47,6 +47,12 @@ use `static mut` variables. How is this possible? This is possible because
 `exception` handlers can *not* be called by software thus reentrancy is not
 possible.
 
+> Note that the `exception` attribute makes transformation of static variables
```

Comment:
This sentence no English. 😅

Reply:
@therealprof I have changed wording. Please, take a look